expression/expression.go

```diff
@@ -1161,6 +1161,13 @@ func canScalarFuncPushDown(scalarFunc *ScalarFunction, pc PbConverter, storeType
 
 	// Check whether this function can be pushed.
 	if !canFuncBePushed(scalarFunc, storeType) {
+		if pc.sc.InExplainStmt {
```

Comment:
for example: select fun1(), agg_fun2() from t  where func3() group by c having fun4(); if funx() cannot be pushed down, will this shows 4 warnings, even though it first encounters func3()?

Reply:
This pr only record the unsupported functions, it does not change the behavior of optimizer, so if `func3` is not supported, and the optimizer decide not pushdown `agg` to the storage layer, then only `func3` will be recorded.

Reply:
other unsupported functions also will be recorded?